### DISCUSSION TITLE
Revoir le rôle du leader

### DISCUSSION THREAD

**Discussion by ElethornGM** [fibreville/minotaure_php#66] [Boite à idées]
- automatiser le fait que quand le leader utilise son pouvoir, il perds le rôle de leader
 - Changer le rôle du leader : trop puissant. à la place : le vote du leader compte comme 10% des voix s'il y a plus de 10 personnages (quelque soit le nombre de votants)

**Comment by uurshin** [comment]
Oui ce système semble générer pas mal de frustration chez les PJs.

Pour le point un, je suis du même avis, mais il faudrait que ce soit une option (et on pourrait mettre l'automatisation par défaut).
Pour le point 2, il faudrait avoir une discussion avec suffisamment de dev / streamers d'ATRPG pour prendre une décision sur le sujet. Ça me parait tout à fait raisonnable, on pourrait en faire une option sélectionnée par défaut également.

**Reply by ElethornGM** [reply]
Pour moi, il ne faudrait même pas que ça automatise l'attribution du Leader, il faudrait juste que ça enlève le rôle du leader (pour ne pas tout le temps avoir un leader actif).
D'accord qu'il faut attendre et en discuter avec la commu si on décide de changer le pouvoir en lui-même.
